### PR TITLE
GnuWin32.Tar version 1.13-1

### PR DESCRIPTION
**Pull request by idleberg**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/68437)

### CODE REVIEW

File: manifests/g/GnuWin32/Tar/1.13-1/GnuWin32.Tar.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: GnuWin32.Tar
+PackageVersion: 1.13-1
```
Comment (ItzLevvie) [inline manifests/g/GnuWin32/Tar/1.13-1/GnuWin32.Tar.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.13
```
This must match Control Panel.

File: manifests/g/GnuWin32/Tar/1.13-1/GnuWin32.Tar.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: GnuWin32.Tar
+PackageVersion: 1.13-1
```
Comment (ItzLevvie) [inline manifests/g/GnuWin32/Tar/1.13-1/GnuWin32.Tar.installer.yaml:5]:
```suggestion
PackageVersion: 1.13
```
This must match Control Panel.

File: manifests/g/GnuWin32/Tar/1.13-1/GnuWin32.Tar.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: GnuWin32.Tar
+PackageVersion: 1.13-1
```
Comment (ItzLevvie) [inline manifests/g/GnuWin32/Tar/1.13-1/GnuWin32.Tar.yaml:5]:
```suggestion
PackageVersion: 1.13
```
This must match Control Panel.

File: manifests/g/GnuWin32/Tar/1.13-1/GnuWin32.Tar.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: GnuWin32.Tar
+PackageVersion: 1.13-1
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://sourceforge.net/projects/gnuwin32/files/tar/1.13-1/tar-1.13-1-bin.exe/download
+  InstallerSha256: 3A85DA1C6611C83739ADCDAD640D76CCB627A3EFFAE408AC1D83BAEF2D69B6E1
```
Comment (ItzLevvie) [inline manifests/g/GnuWin32/Tar/1.13-1/GnuWin32.Tar.installer.yaml:10]:
```suggestion
  InstallerSha256: 3A85DA1C6611C83739ADCDAD640D76CCB627A3EFFAE408AC1D83BAEF2D69B6E1
  AppsAndFeaturesEntries:
  - DisplayName: Tar-1.13 Binaries (GnuWin32)
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/68437#pullrequestreview-1062726493

You forgot to update the version folder to match PackageVersion.

**Comment by idleberg** [comment]
In a previous comment, it was suggested that I change the version from v1.13-1 to v1.13. I'm wondering for the reason behind this, is it wise to ignore a third-party software's version? I'm not too familiar with the GnuWin project, but what if there was an actual version 1.13, then we'd have a conflict.

More importantly, maybe, I'd like to point out that GnuWin32.Wget is currently included with v1.11.4-1.

**Comment by ItzLevvie** [comment]
It's mostly to match Control Panel for better upgrade experience when updating the package but you can also add `DisplayVersion: 1.13` since `1.13-1` is the marketing version used by GnuWin32 Wget.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/68437#issuecomment-1211213620

**Comment by idleberg** [comment]
> Currently WinGet will see `GnuWin32.Tar` as `1.13` from Control Panel and the package contains `1.13-1` so you may end up in an infinity upgrade loop which can possibly (?) be fixed with `DisplayVersion: 1.13`

Thanks for the explanation, but shouldn't it be the other way round, `PackageVersion: 1.13` and `DisplayVersion: 1.13-1`? The examples I looked at seem to do it that way, e.g. https://github.com/microsoft/winget-pkgs/blob/48babb5dfdc8190d04c0b597a2427f6325087dc8/manifests/m/Microsoft/DotNet/Runtime/6/6.0.8/Microsoft.DotNet.Runtime.6.installer.yaml#L19

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Oops - ignore my previous comment above.

Some examples of how it's shown in Control Panel:

`GnuWin32: Wget`:
image

`GnuWin32: Tar`:
image

I think there's two choices here:
- Use `PackageVersion: 1.13` to match Control Panel to prevent upgrade loops. This is how it's done for a lot of packages since 2021.
- [NEW - may cause issues as it was recently introduced] - Use `PackageVersion: 1.13-1` in combination with `DisplayVersion: 1.13` in `AppsAndFeaturesEntries`.

> but shouldn't it be the other way round

No because `DisplayVersion: 1.13-1` won't work here because it doesn't match Control Panel as `AppsAndFeaturesEntries` was created for the application's registry entry in Control Panel.

i.e. `Microsoft.DotNet.Runtime.6` uses `6.0.8.31513` in Control Panel which is why there's `DisplayVersion: 6.0.8.31513` in the manifest.

**Review by ItzLevvie — APPROVED** [review]
I will approve this and fix it up in another PR to test out AppsAndFeaturesEntries since this PR has waiver applied and I don't want to keep this PR behind.

**Pull request merged**